### PR TITLE
Issue #118 

### PR DESCRIPTION
**Pull request by abhisheknalla**
- [ ] Fast Fourier Transformation
- [ ] Sum of squares of first N natural numbers
- [x] Checking for integer overflow on multiplication
- [ ] Convert Radian to Degree
- [x] K-th Primes

Test Cases for: For checking integer overflow on multiplication
Input : a = 100, b = 200 
Output : No

Input : a = 10000000000,
        b = -10000000000
Output : Ye

Test Cases for: For finding k'th prime factor of a number.
Input : n = 225, k = 2        
Output : 3
Prime factors are 3 3 5 5. Second
prime factor is 3.

Input : n = 81, k = 5
Output : -1
Prime factors are 3 3 3 3

These are two very useful math algorithms

### CODE REVIEW

**Comment by prateekiiest** [comment]
This one is really basic, its just a conversion. 
Hence I think it wont much value to the code base, I hope you understand

**Comment by prateekiiest** [comment]
similarly square if n numbers

**Comment by abhisheknalla** [comment]
Ok, I will add more algos. Is the process of pull request correct ?

**Comment by prateekiiest** [comment]
its good @abhisheknalla

**Review by soubh1k — COMMENTED** [review]
This program is not much useful. Maybe you can submit the code for FFT and we will look into it.

**Review by prateekiiest — COMMENTED** [review]
document the code

**Comment by abhisheknalla** [comment]
I am working on the Fast Fourier Transformation Code.

**Comment by abhisheknalla** [comment]
I added README's for the codes. Do you want a more detailed explanation ?

**Comment by prateekiiest** [comment]
lemme see @abhisheknalla

File: Competitive Coding/Math/kthPrimeFactor/kthPrimeFactor.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			return 2;
+	}
+
+	for (int i = 3; i <= sqrt(n); i = i+2)
+	{
+		while (n%i == 0)
+		{
+			if (k == 1)
+				return i;
+
```
Comment (prateekiiest) [inline Competitive Coding/Math/kthPrimeFactor/kthPrimeFactor.cpp:21]: comment all the codes first

**Pull request merged**